css-typed-om/Overview.bs

```diff
@@ -3574,6 +3574,11 @@ which will be filled in with more detail as we go.
 
 : <a>align-content</a>
 ::
+    1. If the specified or computed value is the ''normal'' keyword,
+        return the result of reifying the "normal" keyword.
+    2. If the value is a keyword from <<content-distribution>>,
+        return the result of reifying the keyword value.
+    3. Otherwise, the value is not supported.
```

Comment:
Interesting case with like `<<baseline-position>>`, which can be 'baseline', 'first baseline', 'last baseline'. We can either reify 'baseline' as keyword and the others as unsupported, or treat all three as unsupported in case we promote `<<baseline-position>>` to its own value?

Reply:
We should reify `baseline` and `first baseline` both to the single keyword `baseline`, and treat `last baseline` as unsupported.

Same for `<content-position>`: reify if there's no `<overflow-position>`, unsupported otherwise.